src/login/components/Inputs/CustomInput.js

```diff
@@ -56,7 +56,7 @@ const RenderInput = props => {
         <Fragment key={index}>
           <label key={option[0]} htmlFor={option[1]}>
           <input
-            className={"radio-input"}
+            className={props.meta.error && props.meta.visited ? "radio-input error":"radio-input"}
```

Comment:
I am not sure this condition is working. Upon error, I am able to see the text informing me that "the value is not changed", but I am not able to see a change in class or a change in colour inside the radio button.

Reply:
I have tried and it is working in my environment, can you please check again.